.github/workflows/flame_steering_behaviors.yaml

```diff
@@ -0,0 +1,30 @@
+name: flame_steering_behaviors
+
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/flame_steering_behaviors.yaml"
+      - "packages/flame_steering_behaviors/lib/**"
+      - "packages/flame_steering_behaviors/test/**"
+      - "packages/flame_steering_behaviors/pubspec.yaml"
+  push:
+    branches:
+      - main
+    paths:
+      - ".github/workflows/flame_steering_behaviors.yaml"
+      - "packages/flame_steering_behaviors/lib/**"
+      - "packages/flame_steering_behaviors/test/**"
+      - "packages/flame_steering_behaviors/pubspec.yaml"
+
+jobs:
+  build:
+    uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/flutter_package.yml@v1
+    with:
+      working_directory: packages/flame_steering_behaviors
+      flutter_channel: stable
+      flutter_version: 3.13.5
```

Comment:
👀 shouldn't it be Flutter 3.16?

Reply:
I took it from the `flame_behaviors.yaml` we should update this in a different PR imho

Reply:
Okay with me, before resolving the conversation can you link to an issue that tracks this?

Reply:
Tracked it here: #52